src/licensedcode/cache.py

```diff
@@ -75,12 +78,17 @@ def load_or_build(
         idx_cache_dir = os.path.join(licensedcode_cache_dir, LICENSE_INDEX_DIR)
         create_dir(idx_cache_dir)
         cache_file = os.path.join(idx_cache_dir, LICENSE_INDEX_FILENAME)
+        cached_directories_file = os.path.join(idx_cache_dir, CACHED_DIRECTORIES_FILENAME)
```

Comment:
This was originally used for determining cache invalidation but now that we're changing the approach to explicitly force reindexing this can be deleted.